twitter_tunes/scripts/redis_data.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+import os
+import redis
+import json
+
+
+REDIS_URL = os.environ.get('REDIS_URL')
+
+
+def parse_redis_data(data):
+    """Parse redis data into data dictionary."""
+    try:
+        dat_decoded = data.decode('utf-8')
+        json_string = dat_decoded.replace("'", "\"")
+        json_string = json_string.replace("u\"", "\"")
+        data_dict = json.loads(json_string)
+        return data_dict
```

Comment:
the work you are doing in this function seems a bit suspect to me.  If you intend to decode the data you place in redis as JSON, you should probably put that data _into_ redis as JSON.  Then this work on decoding, replacing chars and so on would be moot.